### PR TITLE
Vast improvements

### PR DESCRIPTION
**Pull request by jamesotoole**

### CODE REVIEW

**Comment by jamesotoole** [comment]
Vast improvements were made.

**Review by saccy — CHANGES_REQUESTED** [review]
if you let my code live side by side with yours i'll allow it

**Comment by jamesotoole** [comment]
Please review

File: nvidia2dollars.ps1 [hunk 1]
```diff
@@ -1,44 +1,44 @@
...
-$JimmyCard = [GPU]::new('gtx1080ti')
-$JimmyCard.Price = 1200
-$JimmyCard.GetCardValue()
-
-$NewCard = [GPU]::new('rtx4090')
-$NewCard.Price = 3500
-$NewCard.GetCardValue()
-
-Compare-Value
+# A poor excuse for comparing values of jimmys current 1080ti to that of a new garbage giant and power hungry 4090
```
Comment (saccy) [inline nvidia2dollars.ps1:1]: excellent work

**Review by saccy — APPROVED** [review]
YOLO LGTM

**Comment by jamesotoole** [comment]
> YOLO LGTM

Please approve though

**Comment by saccy** [comment]
Screenshot 2022-11-24 173837

**Pull request merged**